components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -1006,6 +1010,54 @@ private String interpretQueryString(String queryString, Map<String, String[]> pa
         return queryString;
     }
 
+    /**
+     * To capture the additional authenticator params from the adaptive script and interpret the query string with
+     * additional params.
+     *
+     * @param context Authentication context
+     * @param queryString the query string with additional param
+     * @return interpreted query string
+     */
+    private String getQueryStringWithAuthenticatorParam(AuthenticationContext context, String queryString) {
+
+        Matcher matcher = Pattern.compile(OIDCAuthenticatorConstants.DYNAMIC_AUTH_PARAMS_LOOKUP_REGEX)
+                .matcher(queryString);
+        String value = null;
```

Comment:
What would happen if the value is not provided in the context?

Reply:
if the value is not provided, then the additional param will not be added to the query string.